src/api/operator/utils.ts

```diff
@@ -0,0 +1,54 @@
+// Util functions that only pertain to/deal with operator API related stuff
+
+import BigNumber from 'bignumber.js'
+
+import { ONE_BIG_NUMBER } from 'const'
+
+import { RawOrder } from './types'
+
+export type OrderStatus = 'open' | 'filled' | 'expired' | 'partially filled'
+
+// TODO: move to const or app settings?
+const FILLED_ORDER_EPSILON = '0.0001' // == 0.01%
+
+function isOrderFilled(order: RawOrder): boolean {
+  let amount, executedAmount
+
+  if (order.kind === 'buy') {
+    amount = new BigNumber(order.buyAmount)
+    executedAmount = new BigNumber(order.executedBuyAmount)
+  } else {
+    amount = new BigNumber(order.sellAmount)
+    executedAmount = new BigNumber(order.executedSellAmount)
+  }
+
+  const minimumAmount = amount.multipliedBy(ONE_BIG_NUMBER.minus(FILLED_ORDER_EPSILON))
+
+  return executedAmount.gte(minimumAmount)
+}
+
+function isOrderExpired(order: RawOrder): boolean {
+  return Math.floor(Date.now() / 1000) > order.validTo
+}
+
+function isOrderPartiallyFilled(order: RawOrder): boolean {
+  if (order.kind === 'buy') {
+    return order.executedBuyAmount !== '0'
+  } else {
+    return order.executedSellAmount !== '0'
+  }
+}
+
+export function getOrderStatus(order: RawOrder): OrderStatus {
+  if (isOrderFilled(order)) {
+    return 'filled'
```

Comment:
also here would be nice for `OrderStatus` enum instead of the `type` above though I guess this one is fine due to autocomplete from function retrn type? wdyt @anxolin

Reply:
💯

Reply:
Types do have autocomplete as well 
![screenshot_2021-02-01_10-50-25](https://user-images.githubusercontent.com/43217/106504129-64cf6300-647b-11eb-937c-d921f6503135.png)

Reply:
If it's typed, It's fine for me as it's done here

Reply:
samesies